src/modules/custom.rs

```diff
@@ -34,6 +34,11 @@ pub fn module<'a>(name: &str, context: &'a Context) -> Option<Module<'a>> {
         }
     }
 
+    let repo = context.get_repo();
+    if config.whenrepo && repo.is_err() {
```

Comment:
As it can slow the prompt down, please only try searching for repos if necessary.
```suggestion
    if config.whenrepo && context.get_repo().is_err() {
```